Re-import css/css-view-transitions WPT

nt1m opened the pull request:
#### 07d61079e2b2be309f0433be4b1b6c05ab1ff2df
<pre>
Re-import css/css-view-transitions WPT
<a href="https://bugs.webkit.org/show_bug.cgi?id=276259">https://bugs.webkit.org/show_bug.cgi?id=276259</a>
<a href="https://rdar.apple.com/131182352">rdar://131182352</a>

Reviewed by Aditya Keerthi.

Upstream commit: <a href="https://github.com/web-platform-tests/wpt/commit/f94fd0db0ef5334a69aa0c5b60b266f9c5d2624b">https://github.com/web-platform-tests/wpt/commit/f94fd0db0ef5334a69aa0c5b60b266f9c5d2624b</a>

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/content-escapes-clip-with-abspos-child-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/dialog-in-rtl-iframe-old-expected.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/dialog-in-rtl-iframe-old-ref.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/dialog-in-rtl-iframe-old.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/new-content-ancestor-clipped-2.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/new-content-escapes-clip-with-abspos-child-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/new-content-escapes-clip-with-abspos-child.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/new-content-inline-with-offset-from-containing-block-clipped.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/new-content-root-scrollbar-with-fixed-background-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/old-content-escapes-clip-with-abspos-child-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/old-content-escapes-clip-with-abspos-child.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/old-content-inline-with-offset-from-containing-block-clipped.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/old-content-root-scrollbar-with-fixed-background-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/paint-holding-in-iframe-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/paint-holding-in-iframe-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/paint-holding-in-iframe.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/root-scrollbar-with-fixed-background-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/shadow-part-with-name-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/shadow-part-with-name-nested-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/shadow-part-with-name-nested.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/shadow-part-with-name-overridden-by-important-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/shadow-part-with-name-overridden-by-important.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/shadow-part-with-name.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/sibling-frames-transition-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/sibling-frames-transition-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/sibling-frames-transition.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/support/dialog-in-rtl-iframe-child-old.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/w3c-import.log:

Canonical link: <a href="https://commits.webkit.org/280698@main">https://commits.webkit.org/280698@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/ddd0df0ac26aaf6359de057601146bdfb87616e4

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/57355 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/36683 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/55/builds/9830 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/60977 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/59/builds/7799 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/44307 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/61/builds/7988 "Built successfully") | [  ~~🧪 wpe-wk2~~](https://ews-build.webkit.org/#/builders/34/builds/46445 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [✅ 🧪 wincairo-tests](https://ews-build.webkit.org/#/builders/60/builds/5514 "Passed tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/59385 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/34424 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/49535 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/27308 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/42/builds/31206 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/64/builds/6846 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/6803 "Built successfully") | 
| | [  ~~🧪 api-ios~~](https://ews-build.webkit.org/#/builders/13/builds/53226 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 mac-wk2~~](https://ews-build.webkit.org/#/builders/63/builds/7117 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/62656 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/87/builds/1268 "Built successfully") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/62/builds/7207 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/1/builds/53709 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/86/builds/1274 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/49569 "Passed tests") | [  ~~🧪 api-gtk~~](https://ews-build.webkit.org/#/builders/21/builds/53796 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | 
| [  ~~🛠 🧪 merge~~](https://ews-build.webkit.org/#/builders/19/builds/12677 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/88/builds/1080 "Passed tests") | | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/8555 "Built successfully and passed tests") | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/44/builds/32512 "Built successfully") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/46/builds/33597 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/43/builds/34682 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/33343 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

pxlcoder commented on LayoutTests/TestExpectations:
```diff
@@ -7044,6 +7044,9 @@ imported/w3c/web-platform-tests/css/css-view-transitions/scroller-child.html [ I
 imported/w3c/web-platform-tests/css/css-view-transitions/scroller.html [ ImageOnlyFailure ]
 imported/w3c/web-platform-tests/css/css-view-transitions/iframe-and-main-frame-transition-old-main-new-iframe.html [ ImageOnlyFailure ]
 imported/w3c/web-platform-tests/css/css-view-transitions/iframe-and-main-frame-transition-old-main-old-iframe.html [ ImageOnlyFailure ]
+imported/w3c/web-platform-tests/css/css-view-transitions/paint-holding-in-iframe.html [ ImageOnlyFailure ]
```
No bug for this one?

nt1m replied:
I try to not file bugs until we know what the concrete issue is

nt1m committed:
Re-import css/css-view-transitions WPT
https://bugs.webkit.org/show_bug.cgi?id=276259
rdar://131182352

Reviewed by Aditya Keerthi.

Upstream commit: https://github.com/web-platform-tests/wpt/commit/f94fd0db0ef5334a69aa0c5b60b266f9c5d2624b

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/content-escapes-clip-with-abspos-child-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/dialog-in-rtl-iframe-old-expected.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/dialog-in-rtl-iframe-old-ref.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/dialog-in-rtl-iframe-old.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/new-content-ancestor-clipped-2.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/new-content-escapes-clip-with-abspos-child-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/new-content-escapes-clip-with-abspos-child.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/new-content-inline-with-offset-from-containing-block-clipped.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/new-content-root-scrollbar-with-fixed-background-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/old-content-escapes-clip-with-abspos-child-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/old-content-escapes-clip-with-abspos-child.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/old-content-inline-with-offset-from-containing-block-clipped.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/old-content-root-scrollbar-with-fixed-background-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/paint-holding-in-iframe-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/paint-holding-in-iframe-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/paint-holding-in-iframe.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/root-scrollbar-with-fixed-background-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/shadow-part-with-name-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/shadow-part-with-name-nested-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/shadow-part-with-name-nested.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/shadow-part-with-name-overridden-by-important-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/shadow-part-with-name-overridden-by-important.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/shadow-part-with-name.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/sibling-frames-transition-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/sibling-frames-transition-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/sibling-frames-transition.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/support/dialog-in-rtl-iframe-child-old.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/w3c-import.log:

Canonical link: https://commits.webkit.org/280698@main
